Documentation/ryujit-overview.md

```diff
@@ -99,52 +99,52 @@ A stripped-down dump of the GenTree nodes just after they are imported looks lik
 
 ## Types
 
-The JIT is primarily concerned with “primitive” types, i.e. integers, reference types, pointers, and floating point types.  It must also be concerned with the format of user-defined value types (i.e. struct types derived from System.ValueType) – specifically, their size and the offset of any GC references they contain, so that they can be correctly initialized and copied.  The primitive types are represented in the JIT by the var_types enum, and any additional information required for struct types is obtained from the JIT/EE interface by the use of an opaque CORINFO_CLASS_HANDLE.
+The JIT is primarily concerned with “primitive” types, i.e. integers, reference types, pointers, and floating point types.  It must also be concerned with the format of user-defined value types (i.e. struct types derived from `System.ValueType`) – specifically, their size and the offset of any GC references they contain, so that they can be correctly initialized and copied.  The primitive types are represented in the JIT by the `var_types` enum, and any additional information required for struct types is obtained from the JIT/EE interface by the use of an opaque `CORINFO_CLASS_HANDLE`.
 
 ## Dataflow Information
 
-In order to limit throughput impact, the JIT limits the number of lclVars for which liveness information is computed. These are the tracked lclVars (lvTracked is true), and they are the only candidates for register allocation.
+In order to limit throughput impact, the JIT limits the number of `lclVars` for which liveness information is computed. These are the tracked `lclVars` (`lvTracked` is true), and they are the only candidates for register allocation.
```

Comment:
I would suggest not marking lclVar or lclVars as code, since it is something of a nickname for LclVarDsc (as described above).